tests/functional/test_config_groups.py

```diff
@@ -9,11 +9,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# pylint: disable=redefined-outer-name, unused-argument, duplicate-code, no-self-use
+# pylint: disable=redefined-outer-name, unused-argument, duplicate-code, no-self-use, dangerous-default-value
```

Comment:
it's really should be ignored?

Reply:
it is because https://github.com/arenadata/adcm/pull/1117/files#diff-da45508ed38ce64c9e91a35fc44cc92a8177f36d21e8f71a4dea0c44920f857fR378